node/src/main/scala/coop/rchain/node/node.scala

```diff
@@ -189,6 +184,7 @@ class NodeRuntime(conf: Configuration, host: String)(implicit scheduler: Schedul
                      val prometheusService     = NewPrometheusReporter.service(prometheusReporter)
                      implicit val contextShift = IO.contextShift(scheduler)
                      BlazeBuilder[IO]
+                       .withExecutionContext(Scheduler.singleThread("http-server"))
```

Comment:
Does this mean that the http server is going to have only a single thread at it's disposal?

Reply:
yes, but I think that I'll remove this due to what @goral09 said